### PR TITLE
Re-order Build Tasks to Publish to the Ballerina Central

### PR DESCRIPTION
**Pull request by ThisaruGuruge**
## Purpose
$Subject

### CODE REVIEW

**Review by aashikam — APPROVED** [review]

File: graphql-ballerina/build.gradle [hunk 1]
```diff
@@ -147,8 +149,9 @@ task initializeVariables {
             testParams = "--skip-tests"
         }
 
-        if (graph.hasTask(":${packageName}-ballerina:publish")) {
+        if (graph.hasTask(":${packageName}-ballerina:publish") || graph.hasTask(":${packageName}-ballerina:publishToMavenLocal")) {
```
Comment (BuddhiWathsala) [inline graphql-ballerina/build.gradle:152]: According to this logic, let say, someone has configured the ballerina central token. Then during the local publishToMavenLocal, the module will publish to central. What is the reason for doing that?.

Comment (ThisaruGuruge) [reply graphql-ballerina/build.gradle:152]: We added this to enable publishing locally, when they have configured the access token. (Usually we don't). But I see your point.

Will come up with a solution.

Comment (shafreenAnfar) [reply graphql-ballerina/build.gradle:152]: As you all discussed, I think we don't need to be able to publish to central from local machine. It is always good to do it via github actions.

Comment (ThisaruGuruge) [reply graphql-ballerina/build.gradle:152]: Fixed with #89

**Pull request merged**